iohk-monitoring/src/Cardano/BM/Output/Monitoring.lhs

```diff
@@ -77,6 +82,7 @@ Function |effectuate| is called to pass in a |LogObject| for monitoring.
 instance IsEffectuator Monitor a where
     effectuate monitor item = do
         mon <- readMVar (getMon monitor)
+        effectuate (monBuffer mon) item
```

Comment:
Now I got it! you have a local LogBuffer which records all values that are routed to monitoring.
Then, if the config does not contain a monitor expression for this item, you could avoid putting it into the queue. Or, maybe determine this early in the dispatcher as you do not have access to the configuration here.

Reply:
I did it by analogy with `Editor`: https://github.com/input-output-hk/iohk-monitoring-framework/blob/develop/iohk-monitoring/src/Cardano/BM/Output/Editor.lhs#L128